jest/setupTests.ts

```diff
@@ -1,6 +1,7 @@
 import Adapter from 'enzyme-adapter-react-16';
 import {configure} from 'enzyme';
 import {toHaveNoViolations} from 'jest-axe';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Oh, I SO recommend using these matchers. The errors are so much clearer when tests fail: 

https://github.com/testing-library/jest-dom